### ISSUE TITLE
Amazon Seller Partner (Cloud Only) Connection Error

### ISSUE THREAD

**Issue by misteryeo** [airbytehq/airbyte#14734]
**Please note, this connector is in Alpha. Read more here [on expectations](https://docs.airbyte.com/integrations/).**

We're aware that there is a connection error impacting all Amazon Seller Partner source connectors in Airbyte Cloud.

The team is actively working on resolving this with the Amazon team directly but are facing some challenges with the approval process. Please feel free to follow this issue to be notified of any updates.

**Comment by marcelosantoro** [comment]
Waiting the fix 👍 !!!

**Comment by johnwasserman** [comment]
Does this impact OSS? I ask because the connector doesn't utilize open source

**Comment by animer3009** [comment]
I solved my connection error, but anyway I am not able pull data. getting error:

> 2022-08-26 23:43:26 - Additional Failure Information: The report for stream 'GET_AMAZON_FULFILLED_SHIPMENTS_DATA_GENERAL' was aborted due to a fatal error

**Comment by misteryeo** [comment]
@animer3009 The error you're facing sounds unrelated to this particular issue. Can you please post on https://discuss.airbyte.io/ so the team can help you debug? Thank you!

**Comment by igrankova** [comment]
#13017 Doesn't block this issue anymore.

**Comment by YowanR** [comment]
@davydov-d We tried to set-up a sync on our side and run into this [issue](https://github.com/airbytehq/alpha-beta-issues/issues/257). Could you take a look? Once we solve this, we should be unblocked here :)

**Comment by davydov-d** [comment]
yep I have reproduced that issue. Will try to fix as soon as I have a chance

**Comment by YowanR** [comment]
@YuliiaNahoha This is another issue that felt through the cracks. Not sure why it was not prioritized. Can you see if we can fit this in the sprint, please? 
cc @davydov-d

**Comment by davydov-d** [comment]
I've been working on this issue when it used to be a low priority but then switched to more important tasks and did not complete this one. Attaching a draft PR I've made.

**Comment by YowanR** [comment]
That's fair. Since this is an alpha connector compared to the other higher priority GA connectors that you are working. Thanks for the update! 🙏

**Comment by YowanR** [comment]
@lazebnyi @YuliiaNahoha Let's see if we can fit this into the next sprint as it's impacting customers.

**Comment by animer3009** [comment]
@YowanR 
We are going use this connector for our production system.
If possible, on case changes, please add me in reviewer list.

**Comment by suparagroup** [comment]
great work guys! any idea when this would be put in the pipeline?

**Comment by Blochergroup** [comment]
Hey Guys,
do you have any updates regarding this connector?

This would be really helpful for us!

Thanks!

**Comment by suparagroup** [comment]
> Hey Guys, do you have any updates regarding this connector?
> 
> This would be really helpful for us!
> 
> Thanks!

we are using this connector with no issues on local deployment from our side

**Comment by YowanR** [comment]
Hi everyone! Quick update from the team: We're actively working on this connector and we expect to have this connector up and running again on Airbyte Cloud early January. Thank you for your patience and happy holidays!

**Comment by iamedfyi** [comment]
Is this connector still blocked because of this issue? Could really use it!

**Comment by YowanR** [comment]
The connector is now back online! @iamedfyi Give it a shot and lmk how it goes
